### ISSUE TITLE
ESP8266: PUT/GET runs at 1kB/s via mos.exe + UART

### ISSUE THREAD

**Issue by freedaun** [cesanta/mongoose-os#400]
I did a put of an 80k text file (to an ESP8266), and I thought mos.exe hung. Then I did a get and timed it: the transfer took more than a minute. That's just over 1kB/s, slower than dialup, should be at least 10x faster.

This was the command line: 

    .\mos.exe get big.txt > big.txt --port COM7

Retested with another 4MB NodeMCU board.

**Comment by dimonomid** [comment]
Yeah communication via serial port is slow, at least keep in mind that it goes via multiple JSON frames, so it's encoded as base64 and thus the size is bigger; also the regular random console output goes to the same UART, it also slows it down further.

But I agree there is a room for improvement. No ETA at the moment.

Transfer via other means work much faster, try `mos.exe --port ws://192.168.1.201 get big.txt` (of course replace the IP with the actual IP of your device)
